src/coord/View.ts

```diff
@@ -127,11 +128,13 @@ class View extends Transformable implements CoordinateSystemMaster, CoordinateSy
     /**
      * Set center of view
      */
-    setCenter(centerCoord?: number[]): void {
+    setCenter(centerCoord?: (number | string)[]): void {
         if (!centerCoord) {
             return;
         }
-        this._center = centerCoord;
+        const rect = this.getBoundingRect();
+
+        this._center = [parsePercent(centerCoord[0], rect.width), parsePercent(centerCoord[1], rect.height)];
```

Comment:
The percent value should be relative to the chart width and chart height. Which can be get with `api.getWidth` and `api.getHeight()`.